server/odc-service/src/main/java/com/oceanbase/odc/service/session/factory/DefaultConnectSessionFactory.java

```diff
@@ -104,10 +104,13 @@ private void registerConsoleDataSource(ConnectionSession session) {
                 connectSchema = ConnectionSessionUtil.getUserOrSchemaString(connectSchema, session.getDialectType());
                 ConnectionSessionUtil.setConnectSchema(session, connectSchema);
                 ConnectionSessionUtil.setCurrentSchema(session, connectSchema);
+            } else {
```

Comment:
we may change connect extension to solve this problem?

Reply:
solve it by add userName parameter in `JdbcUrlParser getConnectionInfo(@NonNull String jdbcUrl, String userName) method`